### ISSUE TITLE
File Managing through circles similar to Pinterest pop-up ones

### ISSUE THREAD

**Issue by Ricky1054** [deckerst/aves#1335]
**Is your feature request related to a problem? Please describe.**
Have to move a lot of files to folders. It is quite annoying selecting photos at the slow rhythm of selection, three dots and move to folder.

**Describe the solution you'd like**
In Pinterest, I remember than holding your touch for a sec would make a wheel of circles appear that gave you the options of sharing, deleting, and saving a certain pin into a preferred folder. Something like this in this app I feel might be helpful to manage ones files, having the quick option of sharing, deleting, tagging or copying/moving to a certain folder (preferably within the same screen and not opening a new one to select the folder)

**Describe alternatives you've considered**
Same idea, not a wheel, just holding makes a bunch of circles appear lined below a picture.

**Additional context**
None

**Comment by deckerst** [comment]
There's something that could help you, with the full screen viewer quick actions. The "move" and "edit tags" quick action buttons handle a long press and show recently used folders/tags, so you don't switch to another screen.

**Comment by Ricky1054** [comment]
You are absolutely right, this works like a charm. (Still, the idea of doing this when looking at all the images in a collection/album would be another way of managing quickly files that must go to a different folder than the rest that are surrounding it, whom you can just bulk select.)
